### PR TITLE
Add alert docs

### PR DESCRIPTION
**Pull request by CarolineGard**
This PR includes: 

- Add Alert component to tech-docs
- Documentation for Vue, React and Elements
- Props table for Vue, React and Elements

Still missing: 
- Typography styling, including font size and margins
- Correct color on left border

### CODE REVIEW

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -16,6 +20,14 @@ export default defineConfig({
...
     }
   },
+  vite: {
+    plugins: [
+       uno({
+          presets: [presetWarp({ usePreflight: true })],
+          safelist: classes,
+        }),
+    ]
+  },
```
Comment (AnnaRybkina) [inline docs/.vitepress/config.js:30]: Do we need this really? Where are we using uno styling?

File: docs/.vitepress/theme/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import DefaultTheme from 'vitepress/theme'
 import ApiTable from '../ApiTable.vue'
 import '../bootExamples.js'
 import './custom.css'
+import 'uno.css'
```
Comment (AnnaRybkina) [inline docs/.vitepress/theme/index.js:5]: This can be removed if we won't run vite+uno

File: package.json [hunk 3]
```diff
@@ -7,8 +7,12 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.13",
-    "@warp-ds/vue": "^1.0.0-alpha.2",
+    "@warp-ds/component-classes": "1.0.0-alpha.17",
+    "@warp-ds/elements": "^1.0.0-alpha.8",
+    "@warp-ds/react": "^1.0.0-alpha.9",
+    "@warp-ds/uno": "^1.0.0-alpha.5",
```
Comment (AnnaRybkina) [inline package.json:13]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -7,8 +7,12 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.13",
-    "@warp-ds/vue": "^1.0.0-alpha.2",
+    "@warp-ds/component-classes": "1.0.0-alpha.17",
+    "@warp-ds/elements": "^1.0.0-alpha.8",
```
Comment (AnnaRybkina) [inline package.json:11]:
```suggestion
```

File: package.json [hunk 5]
```diff
@@ -7,8 +7,12 @@
...
+    "@warp-ds/react": "^1.0.0-alpha.9",
```
Comment (AnnaRybkina) [inline package.json:12]:
```suggestion
```

File: package.json [hunk 6]
```diff
@@ -7,8 +7,12 @@
...
+    "@warp-ds/uno": "^1.0.0-alpha.5",
+    "@warp-ds/vue": "^1.0.0-alpha.3",
+    "unocss": "^0.50.1",
```
Comment (AnnaRybkina) [inline package.json:15]:
```suggestion
```

File: docs/.vitepress/config.js (hunk 1 above)
Comment (AnnaRybkina) [reply docs/.vitepress/config.js:30]: We use the styling, I've tested. So disregard this comment

File: package.json (hunk 6 above)
Comment (AnnaRybkina) [inline package.json]:
```suggestion
    "@warp-ds/component-classes": "1.0.0-alpha.17",
    "@warp-ds/uno": "^1.0.0-alpha.5",
    "@warp-ds/vue": "^1.0.0-alpha.3",
    "unocss": "^0.50.1",
```

File: docs/alert/Example.vue [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  const showAlert = true;
+</script>
+
+<template>
+  <h3>Info</h3>
+  <w-alert v-model="showAlert" info title="This is the info variant of the alert element">
+    <p>I am an excellent message for the user.</p>
+  </w-alert>
+  <h3>Positive</h3>
+  <w-alert v-model="showAlert" positive info title="This is the positive variant of the alert element">
```
Comment (magnuh) [inline docs/alert/Example.vue]: `positive` and `info`? 🤔

Comment (magnuh) [reply docs/alert/Example.vue]:
```suggestion
  <w-alert v-model="showAlert" positive title="This is the positive variant of the alert element">
```

File: docs/alert/elements.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+### Usage
+
+```js
+<f-alert variant="info" show="">
+  <p class="font-bold">This is "info" variant of the alert element</p>
+  <p>With an additional description</p>
+</f-alert>
```
Comment (magnuh) [inline docs/alert/elements.md]: Should be `w-alert`?

```suggestion
<w-alert variant="info" show="">
  <p class="font-bold">This is "info" variant of the alert element</p>
  <p>With an additional description</p>
</w-alert>
```

File: docs/alert/vue.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+```js
+import { wAlert } from "@warp-ds/vue";
+```
+
+## Usage
+
+```js
+<f-alert v-model="showAlert" info title="I am a title">
+  <p>I am an excellent message for the user.</p>
+</f-alert>
```
Comment (magnuh) [inline docs/alert/vue.md]:
```suggestion
<w-alert v-model="showAlert" info title="I am a title">
  <p>I am an excellent message for the user.</p>
</w-alert>
```

File: docs/alert/Example.vue (hunk 7 above)
Comment (CarolineGard) [reply docs/alert/Example.vue]: Nice catch

**Pull request merged**